test/functional/feature_signet.py

```diff
@@ -50,14 +68,29 @@ def setup_network(self):
     def run_test(self):
         self.log.info("basic tests using OP_TRUE challenge")
 
+        self.log.info('getblockchaininfo')
+        check_getblockchaininfo(self.nodes[1], self.signets[0])
+        check_getblockchaininfo(self.nodes[2], self.signets[1])
+        check_getblockchaininfo(self.nodes[5], self.signets[2])
+
         self.log.info('getmininginfo')
-        mining_info = self.nodes[0].getmininginfo()
-        assert_equal(mining_info['blocks'], 0)
-        assert_equal(mining_info['chain'], 'signet')
-        assert 'currentblocktx' not in mining_info
-        assert 'currentblockweight' not in mining_info
-        assert_equal(mining_info['networkhashps'], Decimal('0'))
-        assert_equal(mining_info['pooledtx'], 0)
+        # mining info for node on signet 0
```

Comment:
Maybe also add a `check_getmininginfo` helper?

Reply:
Added in edc34044362628f2994cb5fa5de74ca49fce0fd1